ArcadiaParties-SPA/src/app/user/reducers/user-reducer.ts

```diff
@@ -0,0 +1,30 @@
+import { createReducer, on } from '@ngrx/store';
```

Comment:
It is not needed to name file `user-reducer`, because it is the only reducer and it is in user folder.

Reply:
It renamed